CHANGELOG.md

```diff
@@ -49,7 +49,7 @@ We have added partial support for Gateway API, which is not enabled by default,
 Ingress resources can now use all APISIX plugin configurations by setting the annotation `k8s.apisix.apache.org/plugin-config-name=xxx`.
 
 Please try out the release binaries and report any issues at
-https://github.com/apache/apisix-ingress-controller/issues.
+<https://github.com/apache/apisix-ingress-controller/issues>.
```

Comment:
This file is automatically generated by the tool, so there is no need to modify it.

Reply:
This file is modified by the `make update-all` command. Let me see if I can skip it.

Reply:
@tao12345666333 I've fixed it.